### PR TITLE
Fixed Bug #56 "Update SpiltAE internals to snake_case"

### PR DESCRIPTION
**Pull request by Gauravsinghal09**
#### What does this implement/fix? Explain your changes.
Updated the camelCase in SplitAE file to snake_case

### CODE REVIEW

File: mvlearn/embed/splitae.py [hunk 1]
```diff
@@ -175,111 +188,127 @@ def fit(self, Xs, validation_Xs=None):
...
+        ).to(device)
+
+        self.view1_encoder_ = self.view1_encoder_
+        self.view1_decoder_ = self.view1_decoder_
+        self.view2_decoder_ = self.view2_decoder_
+
+        if self.print_graph:
+            print(
+                "Parameter counts: \nview1Encoder: {:,}\nview1Decoder: {:,}"
+                "\nview2Decoder: {:,}".format(
```
Comment (gmischl1) [inline mvlearn/embed/splitae.py:222]: Can you also change this text to snake case, like "view1_encoder" rather than view1Encoder?

**Comment by gmischl1** [comment]
Thanks for the contribution! There are a few lines that are too long for the style guidelines which is why travis-ci failed, but looks pretty good. Once you fix that up we'll get another reviewer to confirm everything.

File: mvlearn/embed/splitae.py (hunk 1 above)
Comment (Gauravsinghal09) [reply mvlearn/embed/splitae.py:222]: Yeah sure, Thanks for pointing out.

**Comment by gmischl1** [comment]
Looks good to me. I'm going to ask another collaborator to take a look and confirm since there are a lot of changes here.

**Pull request merged**